Agents/MackayCalculatorAgent/test/unit/webapp/client.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+from app import app
+
+@pytest.fixture
+def client():
+    """Configures the app for testing
+
+    Sets app config variable ``TESTING`` to ``True``
+
+    :return: App for testing
+    """
+
+    #app.config['TESTING'] = True
```

Comment:
This is highly confusing. This file is in a test folder, so one would assume it is for testing, and the comment strongly suggests that `TESTING` should be set to `True`, yet it is commented out. Please resolve this.

Reply:
Yes this comment-out is uncommented out.

Reply:
I find this is an unused file I forgot to delete and it is deleted in the new commit.